components/sections/NavItem/index.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+import NavItem from './index';
+
+type Story = StoryObj<typeof NavItem>;
+type Meta = MetaObj<typeof NavItem>;
+
+export const Default: Story = {
+  args: {
+    href: '/about',
+    label: 'Learn',
+  },
+};
+
+export const WithExternalLink: Story = {
+  args: {
+    href: 'https://nodejs.org/en',
+    label: 'Learn',
+  },
+};
+
+export const FooterItem: Story = {
+  args: {
+    href: '/about',
```

Comment:
```suggestion
    href: 'https://trademark-policy.openjsf.org/',
```

Reply:
This item is not supposed to be an external link

Reply:
The text is "trademark policy" so I propose to put the  _"correct"_ link

Reply:
This is just a Storybook, Augustin. For all I know the link could point to google.com;